client/src/components/Pages/Education/index.js

```diff
@@ -12,48 +12,45 @@ import { educations, certificates } from './constants';
 
 import './styles.css';
 
-const Education = (props) => {
-  const renderEducations = () => (
+const Education = props => {
+  const renderEducations = () =>
     educations.map(item => (
       <EducationItem
         key={item.id}
         discovered={props[item.nameInStore]}
         {...item}
       />
-    ))
-  );
+    ));
 
-  const renderCertifications = () => (
+  const renderCertifications = () =>
     certificates.map(item => (
       <CertificationItem
         key={item.license}
         discovered={props.certificates}
         {...item}
       />
-    ))
-  );
+    ));
 
   const renderContent = () => (
     <React.Fragment>
       <h3 className="education-section-title">Education</h3>
       <ProgressBar percent={props.discoveryPercent} margintTop />
-      <div className="educations-wrapper">
-        { renderEducations() }
-      </div>
+      <div className="educations-wrapper">{renderEducations()}</div>
       <h3 className="education-section-title">Certifications</h3>
-      <div className="certifications-wrapper">
-        { renderCertifications() }
-      </div>
+      <div className="certifications-wrapper">{renderCertifications()}</div>
     </React.Fragment>
   );
 
   return (
     <div className="education-section-container">
-      {
-        props.discovered
-          ? renderContent()
-          : <EmptyView section="education" suggestion="What education do you have?" />
-      }
+      {props.discovered ? (
+        renderContent()
```

Comment:
Bad fix, I hope this will be fixed with the length increase 🤔

Reply:
This is not caused by the line length 😕
Seems its going to be like this because prettier thing is better 🤷‍♂️